### ISSUE TITLE
handle schema dependencies via python packages (not sure for the title)

### ISSUE THREAD

**Issue by untereiner** [tefra/xsdata#551]
For now the generation searches the xsd recursively and transforms them to python classes.
Would it be possible to say stop to this search but give an import directive instead ?
For example:
- I first generate python classes for a set of xsd files from a targetNamespace called common
- I created a python package for these classes
- Then I want to generate python classes for other xsd files that depends on xsd files from the common namespace

The main reason is: I have multiple standards that share a common background but I don't want to generate everything in a whole python package because the users from one standard do never use the classes from the other one.

Should I generate everything together and separate the files on a later stage ?

**Comment by tefra** [comment]
No it's not possible to stop the process at a certain schema. After the initial parsing the analyzer is trying match types and every absent type (which is valid in xsd) are reset to xs:anySimpleType or xs:anyType.

Which output structure are you using @untereiner, this sounds something like the `namespaces` strategy could help with, It groups classes per target namespace.

**Comment by untereiner** [comment]
I am using the clusters output structure. I had a lot of circular imports (with versions pre 21.6).
But yes I agree the namespace strategy could help with. I'll try again with 21.7. 
Your suggestion if I understand:
- Generate the classes for all xsd files using the namespace strategy
- Separate manually to create different python packages

**Comment by tefra** [comment]
I 've been thinking about make a combination of namespaces+clusters structure style, let me see what I can do, it might be a little tricky.

**Comment by tefra** [comment]
Hey @untereiner 

I just merged a new output structure `namespace-clusters` that I think it would help you and anyone who needs to run the generator for multiple schemas from the same vendor. First of all the namespace style has been updated to convert namespaces to packages similar to jaxb for java.

http://www.travelport.com/schema/universal_v48_0 -> com.travelport.schema.universal_v48_0

- com
  - travelport
    - schema
      -  universal_v48_0

And now it can be combined with the clusters style. Give it a try!
